### PR TITLE
en locale switch warning typo fix

### PR DESCRIPTION
**Pull request by praveenvino39**
Issue: switch_warning has Typo in en.json locale file.
status: Fixed

### CODE REVIEW

**Comment by praveenvino39** [comment]
I have read the CLA Document and I hereby sign the CLA

File: locales/languages/en.json [hunk 1]
```diff
@@ -102,8 +102,6 @@
     },
     "step5": {
       "title": "Explore the Browser",
-      "content1": "You can explore web3 using the browser"
-    },
     "step6": {
```
Comment (gantunesr) [inline locales/languages/en.json:105]: Why this was removed?

**Review by gantunesr — COMMENTED** [review]
Thanks for the contribution @praveenvino39. I left a question

File: locales/languages/en.json (hunk 1 above)
Comment (praveenvino39) [reply locales/languages/en.json]: I was removed by a mistake and I've added it back.

**Pull request closed without merging**